### PR TITLE
merge all

### PR DESCRIPTION
**Pull request by eyadsalama**
@casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2132)

### CODE REVIEW

File: arabic/journey-to-the-uknown-click-here-https/raw.githubusercontent.com/eyadsalama/create-your-own-adventure/patch-1/journey/new journey.md [hunk 1]
```diff
@@ -0,0 +1 @@
+كانت رحلة جميلة قضيتها مع زوجتى فى احدى المنتجعات الرائعة وبينما نحن فى طريقنا للعودة ليلا : اذ يالعربة تتعطل فجأة
```
Comment (casey-collab) [inline arabic/journey-to-the-uknown-click-here-https/raw.githubusercontent.com/eyadsalama/create-your-own-adventure/patch-1/journey/new journey.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: arabic/journey/journey to the unknown.md [hunk 2]
```diff
@@ -0,0 +1 @@
+كانت رحلة جميلة قضيتها مع زوجتى فى احدى المنتجعات الرائعة وبينما نحن فى طريقنا للعودة ليلا : اذ يالعربة تتعطل فجأة
```
Comment (casey-collab) [inline arabic/journey/journey to the unknown.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: arabic/journey/journey to the unknown.md (hunk 2 above)
Comment (eyadsalama) [inline arabic/journey/journey to the unknown.md]: @casey-collab  how to creat a new page ??????

Comment (eyadsalama) [inline arabic/journey/journey to the unknown.md]: i need to add a new directory has a new page to arabic how to do so ??
@cbuckey-uda

**Comment by cbuckey-uda** [comment]
@eyadsalama, I've closed all your pull requests except this one. If you'd like to try to merge a different one instead, feel free to re-open it, but please try to keep only one pull request open at a time.

To create a new page, it needs to be part of the existing story. So you could find something in the existing Arabic story, think of some option for the reader to take, and add a link to a new page. Then in that new page, you should add something that would make sense if the reader chooses that option.

I'm sorry that you're not happy with the current Arabic story, but since I don't speak Arabic I'm not able to clean it up for you. Do you think you could try to add to the existing story even though it's not as adventurous as you would like? You could always add a sudden "twist" of excitement.

**Comment by eyadsalama** [comment]
@cbuckey-uda thanks for your replay 
you said  "and add a link to a new page. Then in that new page, you should add something that would make sense if the reader chooses that option "
i already tried that but whenever i try to add a new page cassey bot said it is not reachable

**Comment by cbuckey-uda** [comment]
@eyadsalama, the reason Casey says the page is not reachable is because there is no link leading to your page. If you add a link in an existing page, and it links to your new page, then the new page will be reachable.

File: arabic/journey/journey to the unknown.md (hunk 2 above)
Comment (kennethmath) [inline arabic/journey/journey to the unknown.md]: Yeah thanks to all my phone was jacked and they took out all my files and put *"-'$*$+. In and I'm lost anyway but they still get in and get everything I do witch ain't much but its still mine and thanks for everyone's help

Comment (kennethmath) [inline arabic/journey/journey to the unknown.md]: What and where that picture come from

Comment (kennethmath) [inline arabic/journey/journey to the unknown.md]: I been trying to translate but it never worked for me

**Comment by kennethmath** [comment]
Actually I scared to do anything for one don't know what I'm doing and secondly if I do something its a a lot of it sorry really but thanks

**Comment by kennethmath** [comment]
SORRY BUT HOW

**Comment by cbuckey-uda** [comment]
@kennethmath I can't understand what you are saying. Is it related to this pull request, or have you commented in the wrong place?

**Comment by cbuckey-uda** [comment]
Closing this pull request in favor of https://github.com/udacity/create-your-own-adventure/pull/2153. If you would still like to address the comments and get this pull request merged, feel free to re-open it.

**Comment by eyadsalama** [comment]
Sent from Yahoo Mail on Android

**Comment by cbuckey-uda** [comment]
@eyadsalama, was your previous comment intentional? I only see my previous comment quoted with no reply, so if you intended to say something it didn't get through.

**Pull request closed without merging**